be/src/block_cache/fb_cachelib.cpp

```diff
@@ -11,22 +11,22 @@ Status FbCacheLib::init(const CacheOptions& options) {
     Cache::Config config;
     config.setCacheSize(options.mem_space_size).setCacheName("default cache").setAccessConfig({25, 10}).validate();
 
-    Cache::NvmCacheConfig nvmConfig;
-    nvmConfig.navyConfig.setBlockSize(4096);
+    if (!options.disk_spaces.empty()) {
```

Comment:
Is it necessary to add a new configuration item? As block_cache_disk_size has no default value, if it is not set or is zero, can we treat the disk cache is disabled?

Reply:
OK. I'll fix that.